### PR TITLE
Converted all ansible tasks to match new schema

### PR DESCRIPTION
**Pull request by yenicapotediaz**

### CODE REVIEW

**Comment by yenicapotediaz** [comment]
This covers ticket https://github.com/samsung-cnct/k2/issues/259

File: ansible/roles/kraken.cluster_common/tasks/main.yaml [hunk 1]
```diff
@@ -6,4 +6,5 @@
   with_items:
     - "{{ kraken_config.clusters }}"
   loop_control:
-    loop_var: a_cluster
\ No newline at end of file
+    loop_var: a_cluster
+  when: a_cluster.providerConfig.type == 'cloudinit'
```
Comment (DStorck) [inline ansible/roles/kraken.cluster_common/tasks/main.yaml]: doesn't a_cluster represent all the clusters? you may not be able to call providerConfig on that

File: ansible/roles/kraken.config/tasks/set-oidc-info.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 - name: Get oidc provider values from cluster services
   set_fact:
     oidc_provider_values: "{{ item.get('values') }}"
-  with_items: "{{ kraken_config.clusterServices.services | default([]) }}"
-  when: item.name == "{{ kraken_config.kubeAuth.authn.oidc.service_name }}"
+  with_items: "{{ a_cluster.clusterServices.services | default([]) }}"
```
Comment (DStorck) [inline ansible/roles/kraken.config/tasks/set-oidc-info.yaml]: clusterServices.services got renamed, i think to helmConfig.charts

Comment (yenicapotediaz) [inline ansible/roles/kraken.config/tasks/set-oidc-info.yaml]: Got it, thanks!

**Comment by coffeepac** [comment]
ok to test

File: ansible/roles/kraken.nodePool/kraken.nodePool.selector/tasks/main.yaml [hunk 3]
```diff
@@ -1,26 +1,8 @@
...
+- include: nodepool-setup.yaml
+  with_items:
     - "{{ kraken_config.clusters }}"
-    - nodePools
   loop_control:
-    loop_var: cluster_node_tuple
-  when: cluster_node_tuple.1.containerConfig.runtime not in ['docker']
\ No newline at end of file
+    loop_var: a_cluster
+  when: (a_cluster.providerConfig.type == 'cloudinit') and (a_cluster.kubeAuth.authn.oidc is defined)
```
Comment (coffeepac) [inline ansible/roles/kraken.nodePool/kraken.nodePool.selector/tasks/main.yaml]: I'm going to revert the changes to this role in a follow-on commit.  Most likely these changes aren't needed.

**Comment by coffeepac** [comment]
there a couple of possible issues with this.  under normal circumstances I would file a review and we would work that path, however I am trying to close this out tonight so we're going to merge and work forward.
